php74.spec

```diff
@@ -139,7 +139,6 @@ BuildRequires: gcc
 BuildRequires: gcc-c++
 BuildRequires: libtool
 BuildRequires: libtool-ltdl-devel
-BuildRequires: pkgconfig(libzip) >= 0.11
```

Comment:
Anyone here have history on why Zip was removed from the build a while back?

Reply:
Yes, the zip extension has moved to a separate package in Fedora so we plan to follow suit.

https://src.fedoraproject.org/rpms/php/pull-request/3

Reply:
Also FYI, I have ported the spec from Fedora for pecl-zip: https://github.com/davidalger/php74-pecl-zip

There are a few others I've updated from their php73 counterparts to rebuild for php74 as well which I intend to share as I can, or when I'm back from my PTO next week.